src/hotspot/share/gc/shenandoah/shenandoah_globals.hpp

```diff
@@ -334,7 +334,7 @@
           "How many times to maximum attempt to flush SATB buffers at the " \
           "end of concurrent marking.")                                     \
                                                                             \
-  product(bool, ShenandoahSuspendibleWorkers, false, EXPERIMENTAL,          \
+  product(bool, ShenandoahSuspendibleWorkers, true, EXPERIMENTAL,           \
```

Comment:
Why this is needed?

Reply:
Any comments on this? Why does it relevant?

Reply:
Ah yes. We must coordinate with concurrent monitor deflation. Specifically, we must prevent monitors from being deflated while we try to get hold of the object mark word for fetching the object size during evacuation. The protocol is relatively simple: relevant threads must respond to safepoint and/or suspension requests.
See also: https://github.com/openjdk/lilliput/blob/47392f37f85747c0e82d21ee475ffa2935c17a0b/src/hotspot/share/runtime/synchronizer.cpp#L1590

Reply:
I meant why it is relevant to this PR? Should this be fixed upstream first?

Reply:
No, involving GCs in the deflation protocol is only really relevant in Lilliput. And this change does it for Shenandoah. I was thinking to remove the flag altogether - not participating in deflation in Lilliput is a correctness problem and we should not even allow that.

Reply:
I see.